Dockerfile.template

```diff
@@ -6,6 +6,7 @@ RUN \
         jq \
         make \
         python3 \
+        py-pip \
```

Comment:
It appears as though `pip` nor `pip3` is installed by default.

Reply:
Good to know!